pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -470,6 +492,73 @@ func (r *Reconciler) manageDbComponent(ctx context.Context, th *v1alpha1.TektonH
 	return nil
 }
 
+func (r *Reconciler) createApiConfigMap(ctx context.Context, th *v1alpha1.TektonHub) error {
```

Comment:
how it will work in case user wants to specify their own catalog but want to use the default categories?